pageTests/trust-admin/hospitals/[id].test.js

```diff
@@ -49,6 +49,7 @@ describe("trust-admin/hospitals/[id]", () => {
       const container = {
         getRetrieveWardsByHospitalId: () => wardsSpy,
         getRetrieveHospitalById: () => hospitalSpy,
+        getRetrieveHospitalVisitTotals: () => jest.fn().mockReturnValue({}),
```

Comment:
Could we add an expectation somewhere to check we're calling this usecase with the trust ID and that it gets set as props?

Reply:
Ye good point 👍